### PR TITLE
Issues/27537 fix depends decorator

### PR DESCRIPTION
**Pull request by DmitryKuzmenko**
Fix for @depends decorator and it's integration test.
For #27537

### CODE REVIEW

**Comment by cachedout** [comment]
Go Go Jenkins!

**Comment by DmitryKuzmenko** [comment]
Go Go Jenkins!

File: tests/integration/files/file/base/_modules/runtests_decorators.py [hunk 1]
```diff
@@ -46,7 +46,7 @@ def missing_depends():
 
 
 @salt.utils.decorators.depends('time', fallback_function=_fallbackfunc)
-def depends_will_fallback():
+def depends_will_not_fallback():
```
Comment (cachedout) [inline tests/integration/files/file/base/_modules/runtests_decorators.py]: This needs a docstring added to make pylint happy.

Comment (DmitryKuzmenko) [inline tests/integration/files/file/base/_modules/runtests_decorators.py]: @cachedout how did you get this? I've just checked, only getting warning about 2 blank lines between functions.

**Comment by cachedout** [comment]
@DmitryKuzmenko I shouldn't have said the linter. It's actually a failing integration test, here: https://jenkins.saltstack.com/job/salt-pr-rs-cent7-n/9146/testReport/junit/integration.modules.sysmod/SysModuleTest/test_valid_docs/

**Comment by DmitryKuzmenko** [comment]
Got it, thank you for pointing.

**Pull request merged**